src/docs/product/performance/display.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: "Display Selection"
+sidebar_order: 2
+description: "The Display dropdown sets the layout of the Performance homepage."
+---
+
+The Display dropdown automatically selects a set of [metrics](../metrics/) and table columns based on the platform of the selected project. For example, selecting a JavaScript project will set the Display to Frontend (Pageload). You can also manually change the Display, but note that some modes may not be applicable. For example, the Web Vitals shown in Frontend (Pageload) mode only reflect pageload transactions.
+
+### Display: All
+
+In Display: All mode, Apdex and TPM are shown. The Apdex threshold can be configured in the project settings.
```

Comment:
I suggest you modify like this throughout this doc to avoid passive voice and add links to explain acronyms for better usability.

```suggestion
Display: All mode shows [Apdex](product/performance/metrics/#apdex) and [TPM](/product/performance/metrics/#throughput-total-tpm-tps). The Apdex threshold can be configured in **[Project] > Settings**.
```